.github/pull_request_template.md

```diff
@@ -0,0 +1,60 @@
+<!--
```

Comment:
How about using raw text instead of comment text?

It seems that most developers don't remove comment in pull request template.
In apache/arrow, we remove comments in our merge script https://github.com/apache/arrow/blob/0c3206772051c52fe2e648623daaca2c4e71cb72/dev/merge_arrow_pr.py#L603-L604 but we don't use merge script in apache/arrow-go. So our commit history have many this comments...

If we use raw text instead of comment text, we can notice it before we use the merge button. We can remove needless texts before we merge.